### ISSUE TITLE
Option "StreamMapping" not implemented in producer.Kinesis

### ISSUE THREAD

**Issue by ppar** [trivago/gollum#155]
`producer/kinesis.go` and `config/kinesis.conf` refer to a "StreamMapping" option, but it's not recognized by the plugin:

```
2017-06-28 16:18:15 CEST | Warning: Unknown configuration key in producer.Kinesis: streammapping
```

The `producer.Kinesis` struct does contain a `streamMap` member that's apparently used to map streams, but it looks like it's not being populated from the config.

**Comment by ppar** [comment]
https://github.com/trivago/gollum/pull/145/commits/4f691c21e1f1568493f6e21f12d57040031ac1ee

**Comment by arnecls** [comment]
The code was in place, the setting was just not being read.
Fixed with commit 9e81987b47b242587a1629822b44750957f08835
